tooling/camel-tooling-util/src/main/java/org/apache/camel/tooling/util/CamelVersionHelper.java

```diff
@@ -32,7 +32,7 @@ private CamelVersionHelper() {
      *
      * @param  base  the base version
      * @param  other the other version
-     * @return       <tt>true</tt> if GE, <tt>false</tt> otherwise
+     * @return       <var>true</var> if GE, <var>false</var> otherwise
```

Comment:
tt is an html tag which is no more valid in HTML 5 which is a requirement for more recent Javadoc version